modules/javafx.controls/src/main/java/javafx/scene/control/ResizeFeaturesBase.java

```diff
@@ -52,6 +52,30 @@ public ResizeFeaturesBase(@NamedArg("column") TableColumnBase<S,?> column, @Name
       this.delta = delta;
   }
 
+  /**
+   * Returns the width of the area available for columns.
+   *
+   * @return the width availabe for columns
+   *
+   * @since 20
+   */
+  public double getContentWidth() {
+      // not available in the base class
+      throw new UnsupportedOperationException("method not available in the base class");
```

Comment:
Since the default implementation of this method throws `UnsupportedOperationException`, I recommend documenting that subclasses must override this method.